cli/golem-cli/tests/app/build_and_deploy_all.rs

```diff
@@ -8,8 +8,20 @@ use test_r::{inherit_test_dep, test};
 inherit_test_dep!(Tracing);
 
 #[test]
-async fn build_and_deploy_all_templates() {
+async fn build_and_deploy_all_templates_default() {
+    build_and_deploy_all_templates(None).await;
+}
+
+#[test]
+async fn build_and_deploy_all_templates_generic() {
```

Comment:
This was already the bottleneck of our CI pipeline, slow enough that I've put the single test in a separate job and tried all kind of tricks to somehow make it as fast as possible. 
So doing it for both app groups sequentially is probably reverting all the CI speed improvements I've achieved. Let's verify this and if needed run this in a separate test group.